### PR TITLE
Rename reference to upstream lib in README

### PR DESCRIPTION
**Pull request by BPScott**
factory_girl has been renamed to factory_bot. Lets reference the new name.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Thanks for the PR. I'm tempted not to, since it was modeled on the original package, with the old name.

**Comment by BPScott** [comment]
But the contents of the package remains exactly the same, with the exception that the name has changed. The name change has resulted in [zero logic changes](https://github.com/thoughtbot/factory_bot/pull/1051). So you're still based on the original packages.

Also what happens if Github ends up breaking their repository redirects? Leaving a link to https://github.com/thoughtbot/factory_girl would break. Even if you disagree with the name change, not pushing users through a 301 redirect would be useful.

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Factory Muffin 3.0
 
 The goal of this package is to enable the rapid creation of objects for the purpose of testing.
 
-It's basically a "[factory\_girl](https://github.com/thoughtbot/factory_girl)", simplified for use with PHP.
+It's basically a "[factory\_bot](https://github.com/thoughtbot/factory_bot)", simplified for use with PHP.
```
Comment (luketlancaster) [inline README.md]: Maybe change this to
```markdown
It's basically a "[factory\_bot](https://github.com/thoughtbot/factory_bot)" (formerly factory_girl), simplified for use with PHP
```
?

I'll admit: when I read the readme my first thought was, "didn't they rename that lib to something else?"

Comment (GrahamCampbell) [reply README.md]: Technically, this package was not based on factory bot though, it was based on "factory girl", as it stood then.

**Pull request closed without merging**